src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    <p>
+      {person.age && (
+        <p className="Person__age">
+          {`I am ${person.age}`}
+        </p>
+      )}
+    </p>
+    <p className="Person__partner">
+      {`${person.partnerName} is my ${person.sex === 'f' ? 'husband' : 'wife'}`}
+      {person.isMarried ? '' : 'I am not married'}
```

Comment:
Consider using only one ternary operator here. Create a variable for husband/wife

Reply:
Hi! I don't know how to pull it off with 1 ternary operator and at the same time not nesting them. I also don't understand in what way I should create a variable for husband/wife and why. So I fixed the code to the best of my understanding.